src/pages-chibi/implementations/anicrush/main.ts

```diff
@@ -0,0 +1,58 @@
+import { PageInterface } from '../../pageInterface';
+
+export const anicrush: PageInterface = {
+  name: 'Anicrush',
+  type: 'anime',
+  domain: 'https://anicrush.to',
+  languages: ['English'],
+  urls: {
+    match: ['*://anicrush.to/*'],
+  },
+  search: 'https://anicrush.to/search?keyword={searchtermPlus}',
+  sync: {
+    isSyncPage($c) {
+      return $c.url().urlPart(3).equals('watch').run();
+    },
+    getTitle($c) {
+      return $c.querySelector('div.main h2').text().trim().run();
+    },
+    getIdentifier($c) {
+      return $c.url().urlPart(4).run();
```

Comment:
make the identifier only the value after the . in the url

Reply:
https://github.com/MALSync/MALSync/blob/master/src/pages-chibi/implementations/bato/mainV3.ts#L68
like that but with last ofc